examples/mirror-demo/index.html

```diff
@@ -47,13 +48,77 @@
     <block type="variables_get"><field name="VAR">i</field></block>
     <block type="variables_get"><field name="VAR">j</field></block>
     <block type="variables_get"><field name="VAR">k</field></block>
-  </xml>
-
+  </xml> -->
+  
   <script>
-    // Inject primary workspace.
+    var toolbox = {
+      "kind": "flyoutToolbox",
+      "contents": [
+        {
+          "kind": "block",
+          "type": "controls_if"
+        },
+        {
+          "kind": "block",
+          "type": "logic_compare"
+        },
+        {
+          "kind": "block",
+          "type": "controls_repeat_ext"
+        },
+        {
+          "kind": "block",
+          "type": "math_number",
+          "fields": {
+            NUM: 123
+          }
+        },
+        {
+          "kind": "block",
+          "type": "math_arithmetic"
+        },
+        {
+          "kind": "block",
+          "type": "text"
+        },
+        {
+          "kind": "block",
+          "type": "text_print"
+        },
+        {
+          "kind": "block",
+          "type": "variables_get",
+          "fields": {
+            "VAR": {
+              "name": "i"
+            }
+          }
+        },
+        {
+          "kind": "block",
+          "type": "variables_get",
+          "fields": {
+            "VAR": {
+              "name": "j"
+            }
+          }
+        },
+        {
+          "kind": "block",
+          "type": "variables_get",
+          "fields": {
+            "VAR": {
+              "name": "k"
+            }
+          }
+        },
+      ]
+    };
+  
+    // // Inject primary workspace. 
```

Comment:
nit: could you remove the extra comment slashes?

Reply:
Done, thank you for catching this!